frontend/src/pages/LegalPage/index.tsx

```diff
@@ -0,0 +1,65 @@
+import React from "react";
+import { useHistory } from "react-router-dom";
+import Button from "../../components/Button";
+import styles from "./style.module.css";
+
+const LegalPage = () => {
+  const browserHistory = useHistory();
+
+  return (
+    <div className={styles.container}>
+      <h1>Legal</h1>
+      <p className={styles.text}>
+        Nerds Oppose Society is a clone of{" "}
```

Comment:
Technically I believe Creative Commons refers to it as "remix" instead of "clone"